std/uni.d

```diff
@@ -8131,6 +8185,7 @@ private auto toCaser(alias indexFn, uint maxIdx, alias tableFn, alias asciiConve
         {
             @property auto save()
             {
+                import std.range.primitives : save;
```

Comment:
The idea is that once module-scope selective imports are allowed in phobos- they're not allowed yet even though the symbol leaking bug is fixed, because downstream users may need some time to update- the import at the top will be made selective and only bring in those five symbols listed there.  So yes, this is redundant for now, but the idea is that it won't be in a couple releases, when the top-level import is made selective.